subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmTestSuite.java

```diff
@@ -104,10 +104,39 @@ public interface JvmTestSuite extends TestSuite, Buildable {
      */
     void useJUnitJupiter(Provider<String> version);
 
+    /**
+     * Use the <a href="https://junit.org/junit5/docs/current/user-guide/">JUnit Platform</a> testing framework.
+     *
+     * <p>
+     *     Gradle will provide the version of JUnit Platform to use. Defaults to version {@code 1.8.2}
+     * </p>
+     *
+     * @since 8.0
+     */
+    void useJUnitPlatform();
```

Comment:
Do we think these methods are useful? Or are users going to get confused with junit platform vs junit jupiter?

Reply:
We verified the preferred term back when we wrote test suites.  It is definitely a little confusing, but more correct.  Ultimate, the goal should be to remove the need for these methods to exist somehow.

Reply:
But this functionality is different in that it does not include the junit jupiter engine. These could be useful for example when running the junit-vintage engine or using some other, perhaps custom, engine

Reply:
I think that's a larger discussion outside the scope of this change.  You should bring it up the next time we do some big picture thinking about suites.

Reply:
Agreed, I'll remove this from the pr